apps/print/backend/src/app.ts

```diff
@@ -261,6 +273,54 @@ export function buildApi(ctx: AppContext) {
       });
     },
 
+    async printAllBallotStyles(input: {
+      languageCode: LanguageCode;
+      ballotType: BallotType;
+      copiesPerStyle: number;
+    }): Promise<void> {
+      const printerStatus = await printer.status();
+      await logger.logAsCurrentRole(LogEventId.PrinterPrintRequest, {
+        message: `Attempting to print all ballot styles with ${input.copiesPerStyle} copies`,
+        ballotProps: JSON.stringify({
+          languageCode: input.languageCode,
+          ballotType: input.ballotType,
+        }),
+        printConnected: printerStatus.connected,
+      });
+
+      const ballots = store.getBallots({
+        languageCode: input.languageCode,
+        ballotType: input.ballotType,
+      });
+
+      let totalPrintCount = 0;
+      for (const ballot of ballots) {
+        for (let i = 0; i < input.copiesPerStyle; i += 1) {
+          await printer.print({
+            data: Buffer.from(ballot.encodedBallot, 'base64'),
+          });
```

Comment:
@nikhilb4a I recommend, instead of calling print for every copy, passing the `copies` or equivalent to `printer.print`. it's better for the printer to manage fewer jobs, and the number here could balloon rapidly